_schemaV2.graphql

```diff
@@ -1385,12 +1385,12 @@ enum ArtworkSorts {
   TITLE_DESC
 }
 
-type ArtworkVersion {
+type ArtworkVersion implements Node {
   # The names for the artists related to this Artwork Version
   artistNames: String
 
   # The artists related to this Artwork Version
-  artists: String
+  artists: [Artist]
```

Comment:
Note that while this is a breaking change to the schema, nothing in our codebase is querying `artists` from `ArtworkVersion` so it should be fine.